actions/hosts.last_checkin.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: hosts.last_checkin
+pack: foreman
+description: List all hosts that haven't reported back 
+runner_type: orquesta
+entry_point: workflows/hosts.last_checkin.yaml
+enabled: true
+parameters:
+  conection:
+    type: string
+    description: "Name of <connection> from this pack's configuration that specifies how to connect to a server."
+    required: false
+  server:
+    required: true
+    type: string
```

Comment:
please add descriptions to the parameters.
for the `time` parameter, see if  you can find a reference in the Foreman docs about how to format time strings and include that in the description